llvm/test/CodeGen/X86/combine-ptest.ll

```diff
@@ -265,7 +265,6 @@ define i32 @ptestz_v2i64_signbits(<2 x i64> %c, i32 %a, i32 %b) {
 ; SSE41-LABEL: ptestz_v2i64_signbits:
 ; SSE41:       # %bb.0:
 ; SSE41-NEXT:    movl %edi, %eax
-; SSE41-NEXT:    pshufd {{.*#+}} xmm0 = xmm0[1,1,3,3]
```

Comment:
This is definitely wrong as we've gone from just needing the signbits of a <2 x i64> to the signbits of a <4  x i32>

Reply:
Err, yeah I had flipped shuffle src/dst in the codes.